commcare_connect/microplanning/views.py

```
@@ -688,3 +699,92 @@ def save_assignment(request, org_slug, opp_id):
+        id=work_area_id,
+        opportunity=request.opportunity,
+        status=WorkAreaStatus.REQUEST_FOR_INACCESSIBLE,
+    )
+    try:
+        inacc_request = (
+            WorkAreaInaccessibilityRequest.objects.select_related("opportunity_access__user")
+            .filter(work_area=work_area)
+            .latest("pk")
+        )
+    except WorkAreaInaccessibilityRequest.DoesNotExist:
+        raise Http404
```

Ruff (0.15.12)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)